In-house patch to remove unnecessary dependencies from cloudbase-init's
requirements file. The specific reasons are as follows:

Babel			Not applicable

eventlet		Not applicable

netaddr			Not applicable

pbr			Downgrade to 0.8.1 or greater

pyserial		Not applicable

comtypes		Not applicable to Solaris (and lack of PEP 426 support)

pywin32			Not applicable to Solaris (and lack of PEP 426 support)

tzlocal			Not applicable to Solaris (and lack of PEP 426 support)

wmi			Not applicable to Solaris (and lack of PEP 426 support)

--- cloudbase-init-0.9.9/requirements.txt.~1~	2015-10-19 06:33:52.000000000 -0700
+++ cloudbase-init-0.9.9/requirements.txt	2016-04-04 19:51:53.527105713 -0700
@@ -1,17 +1,8 @@
-pbr>=1.3
-iso8601
-eventlet
-netaddr>=0.7.6
-pyserial
+pbr>=0.8.1
 oslo.config
 oslo.log
 six>=1.7.0
-Babel>=1.3
 oauthlib
 netifaces
 PyYAML
 requests
-pywin32;sys_platform=="win32"
-comtypes;sys_platform=="win32"
-wmi;sys_platform=="win32"
-tzlocal;sys_platform=="win32"
